src/lib/updateGrantsRecipients.test.js

```diff
@@ -324,6 +324,244 @@ describe('Update grants, program personnel, and recipients', () => {
+
+    // CEO.
+    expect(personnelToAssert[1].role).toBe('ceo');
+    expect(personnelToAssert[1].title).toBe('CEO');
+    expect(personnelToAssert[1].firstName).toBe('F47126');
+    expect(personnelToAssert[1].lastName).toBe('L47126');
+    expect(personnelToAssert[1].prefix).toBe('Ms.');
+    expect(personnelToAssert[1].email).toBe('47126@hsesinfo.org');
+    expect(personnelToAssert[1].active).toBe(true);
+
+    // Policy Council.
+    expect(personnelToAssert[2].role).toBe('policy_council');
+    expect(personnelToAssert[2].title).toBe(null);
+    expect(personnelToAssert[2].firstName).toBe('F47128');
+    expect(personnelToAssert[2].lastName).toBe('L47128');
+    expect(personnelToAssert[2].prefix).toBe('Ms.');
+    expect(personnelToAssert[2].email).toBe('47128@hsesinfo.org');
+    expect(personnelToAssert[2].active).toBe(true);
+
+    // Director.
+    expect(personnelToAssert[3].role).toBe('director');
+    expect(personnelToAssert[3].title).toBe(null);
+    expect(personnelToAssert[3].firstName).toBe('F47124');
+    expect(personnelToAssert[3].lastName).toBe('L47124');
+    expect(personnelToAssert[3].prefix).toBe('Ms.');
+    expect(personnelToAssert[3].email).toBe('47124@hsesinfo.org');
+    expect(personnelToAssert[3].active).toBe(true);
+
+    // Get second program.
+    personnelToAssert = programPersonnelAdded.filter((gp) => gp.programId === 2);
+    expect(personnelToAssert.length).toBe(4);
+
+    // Get third program.
+    personnelToAssert = programPersonnelAdded.filter((gp) => gp.programId === 3);
+    expect(personnelToAssert.length).toBe(4);
+
+    // Get fourth program.
+    personnelToAssert = programPersonnelAdded.filter((gp) => gp.programId === 4);
+    expect(personnelToAssert.length).toBe(6);
+
+    // Filter auth_official_contact deactivated.
+    const authOfficialDeactivated = personnelToAssert.filter((gp) => gp.role === 'auth_official_contact' && gp.active === false);
+    expect(authOfficialDeactivated.length).toBe(2);
+    // Check deactivated ids are what we expect.
+    const ids = authOfficialDeactivated.map((gp) => gp.id);
+    expect(ids).toContain(deactivatedPersonnel.id);
+    expect(ids).toContain(personnelToUpdate.id);
+
+    // Filter auth_official_contact.
+    const authOfficial = personnelToAssert.filter((gp) => gp.role === 'auth_official_contact' && gp.active === true);
+    expect(authOfficial.length).toBe(1);
+
+    // Assert that the old personnel was left alone.
+    const newPersonnel = authOfficial[0];
+    expect(newPersonnel).toBeDefined();
+    expect(newPersonnel.firstName).toBe('F123');
+    expect(newPersonnel.lastName).toBe('L123');
+    expect(newPersonnel.title).toBe('Governing Board Chairperson');
+    expect(newPersonnel.email).toBe('123@example.org');
+    expect(newPersonnel.active).toBe(true);
+    expect(newPersonnel.originalPersonnelId).toBe(personnelToUpdate.id);
+    expect(newPersonnel.effectiveDate).not.toBeNull();
+  });
```

Comment:
Can this test be shortened to only assert 'add if user exists with same name but is deactivated'?